docs/integrations/sources/jotform.md

```diff
@@ -0,0 +1,32 @@
+# Jotform
+Jotform is a powerful online form builder that makes it easy to create robust forms and collect important data.
+
```

Comment:
@topefolorunso can you please add a little snippet explaining how to obtain the API key?